internal/tls/testdata/dummyCa.pem

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
Is it okay to put testdata under /internal? i think it is better to put them under /assets directory.

Reply:
We will create a new package to put all the testdata into it, and create some function to return the absolute path of the test data.
Do you think it is good?

Reply:
Great!